cmake/contract-tools.cmake

```diff
@@ -2,8 +2,8 @@ include(CMakeDependentOption)
 include(CMakeParseArguments)
 include(ExternalProject)
 
-option(BUILD_SYSTEM_CONTRACTS "Build system contracts" ON)
-cmake_dependent_option(BUILD_TEST_CONTRACTS "Build test contracts" ON "BUILD_SYSTEM_CONTRACTS" ON)
+option(BUILD_SYSTEM_CONTRACTS "Build system contracts" OFF)
+cmake_dependent_option(BUILD_TEST_CONTRACTS "Build test contracts" ON "BUILD_SYSTEM_CONTRACTS" OFF)
```

Comment:
I believe it should be
`cmake_dependent_option(BUILD_TEST_CONTRACTS "Build test contracts" OFF "BUILD_SYSTEM_CONTRACTS" OFF)`  

The option is only available when `BUILD_SYSTEM_CONTRACTS` is ON, but either way, it defaults to `OFF`